__tests__/Action.test.ts

```diff
@@ -344,32 +346,37 @@ describe("Action", () => {
 
         await action.perform()
 
-        expect(updateMock).toBeCalledWith(
+        expect(updateMock).toHaveBeenCalledWith(
             id,
             tag,
-            updateBody,
+            generatedReleaseBody,
```

Comment:
Sorry - last thing 😅. Could we add a near-copy of this test which disables the `generateReleaseNotes` input to verify that we properly use the normal body in that case?